foundations/installations/setting_up_git.md

```diff
@@ -197,7 +197,14 @@ Now you need to copy your public SSH key. To do this, we're going to use a comma
 cat ~/.ssh/id_ed25519.pub
 ~~~
 
+**NOTE:** For a legacy system use the following command propmt in the git bash to copy your public SSH key.
+
+~~~bash
+cat ~/.ssh/id_rsa.pub
+~~~
+
 Highlight and copy the output, which starts with `ssh-ed25519` and ends with your email address.
+Highlight and copy the output, which starts with `ssh-rsa` and ends with your email address for legacy users.
 
 Now, go back to GitHub in your browser window and paste the key you copied into the key field. Keep the key type as `Authentication Key` and then, click `Add SSH key`. You're done! You've successfully added your SSH key!
```

Comment:
[Proofreading/nitpick] The ending of this paragraph seems repetitive, so check out my suggestion below to see how you can improve it. 
```suggestion
Now, go back to GitHub in your browser window and paste the key you copied into the key field. Keep the key type as `Authentication Key` and then, click `Add SSH key`. Congrats, you've successfully added your SSH key!
```

Reply:
Thank you for helping me with the pull request, since I'm new here, your help means a lot.